smoke-tests/spec/modsec_spec.rb

```diff
@@ -36,7 +36,7 @@
     before do
       annotations_hash = {'nginx.ingress.kubernetes.io/enable-modsecurity': "true"}
       annotate_ingress(namespace, ingress_name, annotations_hash)
-      sleep 30
+      sleep 10
```

Comment:
Out of curiosity, why the sleep change?

Reply:
Just to clarify: you saw no errors on live-1?

Reply:
The reason I ask is because the API usage between live-1 and everything else is VERY different.

Reply:
This is all on a test cluster. If live turns out to be slower, happy to create another PR to bump back